openapi/tests/openapi_integration/test_group.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from .helpers.helpers import request_with_validation
+from .helpers.collection_setup import basic_collection_setup, drop_collection
+
+collection_name = 'test_collection_reco'
```

Comment:
maybe it would be good to have name which mention grouping

Reply:
Copy-pasted :see_no_evil: